src/rosie.js

```diff
@@ -62,7 +62,30 @@ Factory.prototype = {
     }
 
     builder = typeof value === 'function' ? value : function() { return value; };
-    this.attrs[attr] = { dependencies: dependencies || [], builder: builder };
+    this._attrs[attr] = { dependencies: dependencies || [], builder: builder };
+    return this;
+  },
+
+  /**
+   * Convenience function for defining a set of attributes on this object as
+   * builder functions or static values. If you need to specify dependencies,
+   * use #attr instead.
+   *
+   * For example:
+   *   Factory.define('Person').attrs({
+   *     name: 'Michael',
+   *     age: function() { return Math.random() * 100; }
+   *   })
+   *
+   * @param {object} attributes
+   * @return {Factory}
+   */
+  attrs: function(attributes) {
+    for (var attr in attributes) {
+      if (attributes.hasOwnProperty(attr)) {
```

Comment:
Should have been more specific about this. You should use `Factory.util.hasOwnProp` like in other parts of this file. Not all objects have a `hasOwnProperty` property 😉